package.json

```diff
@@ -82,6 +82,8 @@
     "typescript": "^2.5.2"
   },
   "dependencies": {
+    "@types/progress": "^2.0.1",
+    "appcenter-file-upload-client": "^0.0.6",
```

Comment:
Hi @buptkang, please change package version to `0.0.5`, I believe that `>= 0.0.6` is not going to work on Windows due to incorrect superagent dependency. `package.json` of  `appcenter-file-upload-client` should be updated to use the stable version of superagent.

Please let me know if you have any questions.

Reply:
Thanks for this information.

Reply:
Switched back to the previous version.